frontend/apps/web/src/pages/all-tests/index.tsx

```diff
@@ -0,0 +1,285 @@
+              variant="outline"
+              size="medium"
+            >
+              All Behaviours
+            </Button>
+          </Buttons>
+        </HeaderWrapper>
+      </PageLayout.Header>
+    )
+  }
+
+  const prepareAllTestsChart = () => {
+    return Object.entries(
+      allTests.reduce(
+        (d, test) => {
+          if (test.kind === 'e2e') {
+            d.e2e += 1
+          } else if (test.kind === 'benchmark') {
+            d.benchmark += 1
+          } else if (test.kind === 'api') {
+            d.api += 1
+          } else if (test.kind === 'integration') {
+            d.api += 1
+          } else if (test.kind === 'unit') {
+            d.api += 1
+          }
+          return d
+        },
+        {
+          e2e: 0,
+          benchmark: 0,
+          api: 0,
+          integration: 0,
+          unit: 0,
+        },
+      ),
+    ).map(([key, count]) => ({
+      name: key,
+      percentage: (count / allTests.length) * 100,
+    }))
+  }
+
+  const allTestsKinds = prepareAllTestsChart()
+
+  const prepareAllTestsStatus = () => {
+    return Object.entries(
+      allTests.reduce(
+        (d, test) => {
+          if (test.status === 'pass') {
+            d.pass += 1
+          } else if (test.status === 'fail') {
+            d.fail += 1
+          } else {
+            d.missing += 1
+          }
+          return d
+        },
+        {
+          pass: 0,
+          fail: 0,
+          missing: 0,
+        },
+      ),
+    ).map(([key, count]) => ({
+      name: key,
+      percentage: (count / allTests.length) * 100,
+    }))
+  }
+  const allTestsStatus = prepareAllTestsStatus()
+  const pageLayout = usePageLayout({
+    header: <Header />,
+    footer: <PageLayout.Footer />,
+  })
+  return (
+    <PageLayout {...pageLayout}>
+      <PageLayout.Section>
+        <BarWrapper>
+          <Text type="text xl">All Tests Kinds</Text>
+          <ProgressBar
+            legend
+            onClick={() => navigate('/repository-details')}
+            data={allTestsKinds.map(({ name, percentage }) => ({
+              name,
+              percentage,
+            }))}
+          />
+        </BarWrapper>
+        <BarWrapper>
+          <Text type="text xl">All Tests Status</Text>
+          <ProgressBar
+            legend
+            onClick={() => navigate('/repository-details')}
+            data={allTestsStatus.map(({ name, percentage }) => ({
+              name,
+              percentage,
+            }))}
+          />
+        </BarWrapper>
+        <Heading>
+          <Text type="text xl">List of all tests ({allTests.length})</Text>
+        </Heading>
+        <Table
+          variant="default"
+          columns={{
+            suite: {
+              header: 'Suite',
+              width: 220,
+              Cell: ({ data }) => {
+                return (
+                  <>
+                    <Text type="text s">
+                      {data.id.split('/')[0]}
```

Comment:
Can you show me why there is a split here?

Reply:
Id is single string and i need to represent it like this
![image](https://user-images.githubusercontent.com/85948457/143398857-84adfc09-56a8-4f63-b448-3282ab3ba356.png)

Reply:
we have path and functionName, maybe it's better to use that.

Reply:
Done